### DISCUSSION TITLE
Nextjs Router /_next/static/…/pages/***.js 404 Not Found

### DISCUSSION THREAD

**Discussion by saouas** [vercel/next.js#12947] [Help] [answered]
Hello Everyone,

I am building a chat web app with Nextjs and Socket io on frontend part.
My project totallty work in dev mode (when i npm run dev).

So I decided to put in production my work , backend is on a personal VM instance and for deploying frontend i use ZEIT/NOW (Vercel).

But i have a problem with the frontend part , all the link are unreachable in production mode and i got these error : "/_next/static/…/pages/***.js 404 Not Found". It seems that router doesn't work.

_ps : I also try simple to next build and test it in a production mode and i got same error as Zeit deployment._

Here is my code to handle Routing using Router from 'next/router'
`const handleJoin = () => {
        (name && room) ? (Router.push({
            pathname:`/chat`,
            query: {
                name: name,
                room: room
            },})) : e.preventDefault()
    }`

**How to reproduce ?**
1 - Go on https://frontend-chatchat.now.sh/
2 - Fill name and room
3 - Click on "ENTRER"
4 - Tadam look at the console 😞

Please can someone help me with this case ?

**Comment by ijjk** [comment, marked as answer]
Hi, it looks like you named your page `Chat.js` with a capital `C` although you are navigating to `chat` with a lower case `c`. This is causing the page to 404 when visited in production since it is case-sensitive. Notice visiting https://frontend-chatchat.now.sh/Chat works properly

**Reply by saouas** [reply]
That was it. 
Thank's you very much for you answer.
I was however sure that it isn't case-sensitive -_-

**Comment by dpkpaa** [comment]
same but for dynamic pages only i am using next-routes

**Reply by gaurav5430** [reply]
i have a similar problem when using next-routes
@dpkpaa did you get any insights on this?
